### PR TITLE
Add links to code on GitHub

### LINKED ISSUE #519: Add links to code from documentation?
When I link to http://googlecloudplatform.github.io/gcloud-node/#/docs/v0.13.2/pubsub/subscription?method=ack I was wondering how to get from that method to reading the code for that method.

Is there any way we can add a link that brings us to the actual code on Github...?

### PR DESCRIPTION
**Pull request by ryanseys**
Fixes #519

My angular-foo is not the best so @stephenplusplus should definitely not hold back in telling me if this is a terrible way to do it. :)

### CODE REVIEW

**Comment by ryanseys** [comment]
Sorry I would make a link to my Github Pages so you can preview it but I can't figure out how to replicate the docs folder properly over to the gh-pages branch. :blush:

File: docs/site/components/docs/docs.js [hunk 1]
```diff
@@ -149,6 +149,8 @@ angular
                 }),
               description: $sce.trustAsHtml(
                   formatHtml(detectLinks(detectModules(obj.description.full)))),
+              lineNum: obj.codeStart,
+              lineNumLink: obj.codeStart ? '#L' + obj.codeStart : '',
```
Comment (stephenplusplus) [inline docs/site/components/docs/docs.js]: Super impressed you were able to work your way through this file. I hope to improve the maintainability of the site soon.

**Comment by stephenplusplus** [comment]
My process isn't very elegant, but:

1. Clone another gcloud-node (gcloud-node-gh-pages), which will always sit on the gh-pages branch
2. `cd to/your/gcloud-node`
3. `npm run docs`
4. `cp -R docs/* ../gcloud-node-gh-pages`

:dancer:

File: docs/site/components/docs/docs.js (hunk 1 above)
Comment (ryanseys) [inline docs/site/components/docs/docs.js]: Yeah it was a little daunting at first but luckily there's enough code around that I could kinda just copy what already existed and re-shape for my needs.

**Comment by ryanseys** [comment]
Oh yeah, that's probably smarter to have two repos. gh-pages is a nice idea in theory but most of the time I wish it was its own repo like wikis are.

**Comment by stephenplusplus** [comment]
Oh my gosh, it's perfect.

**Comment by ryanseys** [comment]
:blush:

**Pull request merged**